sr_ur_arm_calibration_loader/src/sr_ur_arm_calibration_loader/sr_ur_arm_calibration_loader.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/python
+
+# Copyright 2019 Shadow Robot Company Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the Free
+# Software Foundation version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program. If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import rospy
+import roslaunch
+import rospkg
+import Tkinter as tk
+import tkMessageBox as messageBox
+import paramiko
+import sys
+import yaml
+from rosparam import upload_params
+
+
+class SrUrLoadCalibration(object):
+    def __init__(self, arm_info_in=[]):
+        self._default_kinematics_config = ''
+        self._arm_calibrations_folder = ''
+        if [] == arm_info_in:
+            rospy.logerr("No arms specified, cannot find arm calibration")
+        self._arm_info_in = arm_info_in
+        CONST_ARM_TYPE = self._arm_info_in[0]['arm_type'].lower()
```

Comment:
It'll set arm type from the first arm specified (right). At the moment it can't handle bimanual with different arm types. Do you think I should make it work for this case?